frontend/src/tests/mocks/dataMocks.js

```diff
@@ -455,6 +435,14 @@ export const REGION_OPTIONS = {
   }
 ]};
 
+export const REGION_HASH = {
+  "SW": "South West",
```

Comment:
I think the quotes are unnecessary here. Airbnb linting rules requests removing them if they're not needed